src/components/UsageReportConsent.js

```diff
@@ -0,0 +1,50 @@
+import { h } from 'preact'
+import { useState, useRef, useEffect } from 'preact/hooks'
+import { useNavigation, useI18n, useSoftkey } from 'hooks'
+import { RadioListView } from 'components'
+import { getConsentStatus, setConsentStatus } from 'utils'
+
+export const UsageReportConsent = ({ close }) => {
+  const i18n = useI18n()
+  const containerRef = useRef()
+  const listRef = useRef()
+  const hasConsented = getConsentStatus()
+  const [items, setItems] = useState([
+    { title: i18n('usage-consent-yes'), isSelected: hasConsented },
+    { title: i18n('usage-consent-no'), isSelected: !hasConsented }
+  ])
+  const [, setNavigation, getCurrent] = useNavigation('UsageReportConsent', containerRef, listRef, 'y')
+
+  const onKeyCenter = () => {
+    const { index } = getCurrent()
+    setItems(items.map((item, i) => {
+      item.isSelected = (i === index)
+      return item
+    }))
```

Comment:
Refactor idea :

the `isSelected` attribute needs to reset every time having an action, maybe this can be wrapped in the listview/radiolistview components, so we don't have to define the settings in all the components.